src/commands/apply.js

```diff
@@ -10,6 +10,7 @@ class ApplyCommand extends TerraformCommand {
    */
   configure() {
     this
+      .enableElasticSearchLogging()
```

Comment:
I believe this function must be included into `terraform-command.js` instead of in each `[Placeholder]Command` ... @euliancom what do you think?

Reply:
Updated